web/src/utils/local-storage-hook.ts

```diff
@@ -0,0 +1,74 @@
+import _ from 'lodash';
+import * as React from 'react';
+
+export const LOCAL_STORAGE_PLUGIN_KEY = 'network-observability-plugin-settings';
+export const LOCAL_STORAGE_COLS_KEY = 'netflow-traffic-columns';
+export const LOCAL_STORAGE_REFRESH_KEY = 'netflow-traffic-refresh';
+export const LOCAL_STORAGE_SIZE_KEY = 'netflow-traffic-size';
```

Comment:
I wonder if having separate root keys could be simpler, than a single root key + json blob with sub-keys .. but not really sure about that ; so let's keep with it :-)

Reply:
I started with separated and saw that the console use a unique key so I guess using a single one for the plugin is consistent